client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts

```diff
@@ -54,6 +54,7 @@ export function getEnhancedTasks(
 				case 'plan_selected':
 					taskData = {
 						title: translate( 'Choose a Plan' ),
+						disabled: true,
```

Comment:
This affects all flows using launchpad, including newsletters. For newsletters customers would always have selected a plan (and paid for it if necessary), so it was marked as completed. It wasn't marked as disabled though so users were able to go back and change it (e.g. to upgrade from free to a paid plan)